scapy/layers/bluetooth.py

```diff
@@ -1038,9 +1038,52 @@ class HCI_Cmd_Write_LE_Host_Support(Packet):
 
 
 # 7.4 INFORMATIONAL PARAMETERS, the OGF is defined as 0x04
+class HCI_Cmd_Read_Local_Version_Information(Packet):
+    """
+
```

Comment:
In previous PRs the class docstring was added but without newlines. Should the empty lines be removed?